selfify/content_scripts/selfify.js

```diff
@@ -0,0 +1,41 @@
+/*
+"Selfifies" the web page by:
+* Removing every node in the document.body
+* Inserting the selected image
+* Removing itself as a listener
+*/
+function selfify(request, sender, sendResponse) {
+  debugger;
+  removeEverything();
+  insertImage(request.imageURL, request.fileName);
+  browser.runtime.onMessage.removeListener(selfify);
+}
+
+/*
+Remove every node under document.body
+*/
+function removeEverything() {
+  while (document.body.firstChild) {
+    document.body.firstChild.remove();
+  }
+}
+
+/*
+Given a URL to an image, create and style an iframe containing an 
+IMG node pointing to that image, then insert the node into the document.
+*/
+function insertImage(imageURL, fileName) {
+  var insertImage = document.createElement("iframe");
+  insertImage.setAttribute("src", browser.extension.getURL(`/viewer.html?blobURL=${imageURL}`));
+  insertImage.setAttribute("style", "width: 100vw");
+  insertImage.setAttribute("style", "height: 100vh");
```

Comment:
This overwrites line 30, so you end up only setting height, which I think might be why the image ends up tiny.

I think you need something more like:

`insertImage.setAttribute("style", "width: 100vw; height: 100vh;");`